getting-started/download.md

```diff
@@ -1,43 +1,70 @@
 ---
-description: Downloading the WolvenKit application or Console
+description: Downloading the WolvenKit application
 ---
 
 # Download
 
-WolvenKit is available from the following locations:
-
-{% embed url="https://github.com/WolvenKit/WolvenKit/releases" %}
-
-| Package                                                                       | Latest Release                                                                                                                                                                              | Checks                                                                                                         |
-| ----------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------------------------------------- |
-| [WolvenKit Nightly](https://github.com/WolvenKit/WolvenKit-nightly-releases/) | [![GitHub release (latest by date)](https://img.shields.io/github/v/release/WolvenKit/WolvenKit-nightly-releases)](https://github.com/WolvenKit/WolvenKit-nightly-releases/releases/latest) | ![GitHub Workflow Status](https://img.shields.io/github/workflow/status/WolvenKit/WolvenKit/WolvenKit-Nightly) |
-| [WolvenKit](https://github.com/WolvenKit/WolvenKit/)                          | [![GitHub release (latest by date)](https://img.shields.io/github/v/release/WolvenKit/WolvenKit)](https://github.com/WolvenKit/WolvenKit/releases/latest)                                   | ![GitHub branch checks state](https://img.shields.io/github/workflow/status/WolvenKit/WolvenKit/check-only)    |
-
-The latest GitHub WolvenKit release will be marked on the left column with green text. Expand the **Assets** tab to view available downloads for any given WolvenKit release.
+The recommended download is the WolvenKit Nightly installer
```

Comment:
The WolvenKit nightly build does not post an installer version. Additionally, it doesn't appear WolvenKit has had any installer release since 8.4.3.

Also while the current release doesn't support game version 1.6, generally speaking stable releases are preferred.

Reply:
> Nightly build does not post an installer
Unless someone is familiar with GitHub, the download table is very confusing. I tried making it a proper table that had headers for every column, but it looked really ugly. The current look is pretty, but the Package links look like they go to the zipped program, and the link-images look like they're just pictures of what the Package versions are. I agree that the ZIP is not technically an installer, but it ultimately removes confusion on the difference between the Source Package and the ZIP file (zip of what? the source, the program, other resources?).

> generally speaking stable releases are preferred
I totally agree! What can be done to get the stable version updated more often? Right now it straight up doesn't work and also the stable version's download page says, "❗INCOMPATABLE WITH GAME VERSION 1.6+ ❗Use WolvenKit development build 8.7 or higher for use with Cyberpunk game version 1.6".